samples/net/https_client/src/main.c

```diff
@@ -17,23 +17,21 @@
 #endif
 
 #define HTTPS_PORT		"443"
-
 #define HTTPS_HOSTNAME		"example.com"
-
 #define HTTP_HEAD                                                                               \
 				"HEAD / HTTP/1.1\r\n"                     			\
 				"Host: " HTTPS_HOSTNAME ":" HTTPS_PORT "\r\n"			\
 				"Connection: close\r\n\r\n"
 
 #define HTTP_HEAD_LEN		(sizeof(HTTP_HEAD) - 1)
-
 #define HTTP_HDR_END		"\r\n\r\n"
 
 #define RECV_BUF_SIZE		2048
 #define TLS_SEC_TAG		42
 
 /* Macros used to subscribe to specific Zephyr NET management events. */
 #define L4_EVENT_MASK		(NET_EVENT_L4_CONNECTED | NET_EVENT_L4_DISCONNECTED)
+#define IP_EVENT_MASK		(NET_EVENT_IPV4_ADDR_ADD | NET_EVENT_IPV6_ADDR_ADD | NET_EVENT_IPV6_CMD_DAD_SUCCEED)
```

Comment:
`NET_EVENT_IPV6_CMD_DAD_SUCCEED` -> `NET_EVENT_IPV6_DAD_SUCCEED`
Is it really needed though? Doesn't seem to be handled in `ip_event_handler()`.